test-tool/test_async_write.c

```diff
@@ -113,7 +201,23 @@ test_async_write(void)
 
 		ret = iscsi_service(sd->iscsi_ctx, pfd.revents);
 		CU_ASSERT_EQUAL(ret, 0);
+
+		/* attempt logout after one of the dispatch IOs has completed */
+		if (!state.logout_sent && (state.io_completed > 0)) {
```

Comment:
Can the parentheses be left out from this expression?

Reply:
Will do.